ProfessorAssessor/Networking/Networker.m

```diff
@@ -37,34 +39,100 @@ + (void)createCourseWithName:(NSString *)name
     [newCourse saveInBackgroundWithBlock:completion];
 }
 
-+ (void)buildReview:(Professor *)professor
-         withCourse:(Course *)course
-        withContent:(NSString *)content
-         withRating:(NSNumber *)rating
-     withCompletion:(PFBooleanResultBlock)completion {
-    // TODO: check if you can create a professor
-    // TODO: check if you can create a course
-    // TODO: make sure content isn't empty
-    // TODO: if all the validation passes, create a new Review object
-    
++ (void)updateDatabaseForNewReviewWithProfessor:(Professor *)professor
+                                         course:(Course *)course
+                                        content:(NSString *)content
+                                         rating:(NSNumber *)rating
+                                     completion:(void(^)(BOOL succeeded,
+                                                         NSError *_Nullable error))completion {
+    Review *newReview = [Networker createReviewWithCourse:course
+                                                professor:professor
+                                                  content:content
+                                                   rating:rating];
+
+    PFQuery *query = [Review query];
+
+    [query whereKey:@"professor" equalTo:professor];
+
+    [newReview saveInBackgroundWithBlock:^(BOOL succeeded,
+                                           NSError *_Nullable error) {
+        if (error == nil) {
+            [query
```

Comment:
Having to fetch to save seems inefficient. Does it make sense to preload / cache this data?